example/src/classes/util/element/element-builder.js

```diff
@@ -0,0 +1,23 @@
+import DefaultBuilder from '../default-builder';
+
+/**
+ * Пример наследования классов.
+ * Данная реализация строителя, унаследованная от базового строителя, добавляет дополнительный
+ * собственный метод addInnerElement, а все остальные методы родительского класса использует
+ * без изменений.
+ */
+export default class ElementBuilder extends DefaultBuilder {
+    /**
+     * Добавление дочерними классами нового метода расширяет функциональность родительского.
+     * @param {HTMLElement | DefaultBuilder} element
+     * @returns {ElementBuilder}
+     */
+    addInnerElement(element) {
```

Comment:
This method should be in the base class for components, this class should be deleted.